doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -4412,119 +4412,179 @@ Material:AirGap,B1 - AIRSPACE RESISTANCE,  ! Material Name
+    0.066,               !- Moisture Equation Coefficient a {dimensionless}
+    1,                   !- Moisture Equation Coefficient b {dimensionless}
+    0,                   !- Moisture Equation Coefficient c {dimensionless}
+    1,                   !- Moisture Equation Coefficient d {dimensionless}
+    0.007,              !- Surface-layer penetration depth {m}
+    0.024,               !- Deep-layer penetration depth {m}
+    0,                   !- Coating layer thickness {m}
+    1;                   !- Coating layer water vapor diffusion resistance factor {dimensionless}
+```
+
+This set of inputs is for gypsum board with density 750 kg/m^3. This also assumes 2 coats of latex paint:
+
+```idf
+MaterialProperty:MoisturePenetrationDepth:Settings,
+    Concrete,            !- Name
+    6.0,                 !- Water Vapor Diffusion Resistance Factor {dimensionless}
+    0.0065,              !- Moisture Equation Coefficient a {dimensionless}
+    0.65,                !- Moisture Equation Coefficient b {dimensionless}
+    0.022,               !- Moisture Equation Coefficient c {dimensionless}
+    10,                  !- Moisture Equation Coefficient d {dimensionless}
+    0.021,               !- Surface-layer penetration depth {m}
+    0.08,                !- Deep-layer penetration depth {m}
+    0.0003,              !- Coating layer thickness {m}
+    6000;                !- Coating layer water vapor diffusion resistance factor {dimensionless}
+```
+
+Finally, here are values representing the empirical whole-house inputs from Woods et al., 2014 (see Engineering Reference). Density is kg/m^3:
+
+```idf
+MaterialProperty:MoisturePenetrationDepth:Settings,
+    Concrete,            !- Name
+    8.0,                 !- Water Vapor Diffusion Resistance Factor {dimensionless}
+    0.012,               !- Moisture Equation Coefficient a {dimensionless}
+    1,                   !- Moisture Equation Coefficient b {dimensionless}
+    0,                   !- Moisture Equation Coefficient c {dimensionless}
+    1,                   !- Moisture Equation Coefficient d {dimensionless}
+    0.019,               !- Surface-layer penetration depth {m}
+    0.113,               !- Deep-layer penetration depth {m}
+    0,                   !- Coating layer thickness {m}
+    1;                   !- Coating layer water vapor diffusion resistance factor {dimensionless}
 ```
 
+Other materials inputs can be estimated using the equations above and material properties from a variety of sources, such as [*Kumaran, 1996*](http://www.iea-ebc.org/fileadmin/user_upload/images/Pictures/EBC_Annex_24_Report_3.pdf), the [*WUFI*](http://WUFI.de/en/) simulation software, or the [*ASHRAE 1018-RP*](http://www.techstreet.com/products/1719052) report.
 
 ### Moisture Penetration Depth (EMPD) Outputs
 
-Output variables applicable to heat transfer surfaces using EMPD model:
+#### EMPD surface inside face water vapor density [kg/m^3]
```

Comment:
No superscript ^ symbols here and below in the headings, simply [kg/m3] so it matches the source and actual output.